packages/x-data-grid/src/components/columnsManagement/GridColumnsManagement.tsx

```diff
@@ -254,6 +255,7 @@ function GridColumnsManagement(props: GridColumnsManagementProps) {
                   }
                   tabIndex={-1}
                   onClick={handleSearchReset}
+                  edge="end"
```

Comment:
Could we make this spacing consistent too?

![SCR-20250218-oels](https://github.com/user-attachments/assets/0fea2018-d919-4508-a65b-d47c7bacd0a0)

Reply:
Good spot. It looks like having an `IconButton` with `size="small` as an input adornment is problematic as it doesn't get positioned correctly. Using a regular sized icon button results in this:

<img width="283" alt="Screenshot 2025-02-18 at 14 10 20" src="https://github.com/user-attachments/assets/da8e749e-1b84-4ae9-810a-f8962aa1f9b5" />

The spacing on the end adornment is visually closer to the start adornment's spacing:

<img width="279" alt="Screenshot 2025-02-18 at 14 11 21" src="https://github.com/user-attachments/assets/00225a69-129d-4d8c-a704-f489c2e3663d" />